.github/renovate.json5

```diff
@@ -282,6 +282,19 @@
         "v1.13"
       ]
     },
+    {
+      // Do not upgrade etcd as they have removed bash starting from v3.5.5
+      "enabled": false,
+      "matchDepNames": [
+        "gcr.io/etcd-development/etcd",
```

Comment:
I would not include `"gcr.io/etcd-development/etcd"`, which is only used in GH actions, and has already bumped to newer versions: https://github.com/cilium/cilium/blob/a7f1f6753c5c67dbcd26d68577d2973ff50fbe38/.github/actions/kvstore/action.yaml#L10